website/sidebars.json

```diff
@@ -27,7 +27,8 @@
 									"install/arm/activating-a-validator"
 								],
 
-		"Running Prysm": 	[	"prysm-usage/deploy-eth1-node",
+		"Running Prysm": 	[	"prysm-usage/get-more-peers",
+								"prysm-usage/deploy-eth1-node",
```

Comment:
I think it would be better just above the FAQ section as it's an optional page with advanced information not required for all setups

Reply:
done